README.md

```diff
@@ -17,6 +17,14 @@ juju deploy vault-k8s -n 5 --trust
 
 We recommend deploying Vault with an odd number of units.
 
+### Integrate with Ingress
```

Comment:
Have you tested the docs here in the README.md with vault behind traefik?

Reply:
Yes, I have tested.